spec/controllers/checkout_controller_spec.rb

```diff
@@ -308,6 +308,62 @@
         end
       end
 
+      context "with no payment source" do
```

Comment:
FYI: you referenced a Bugsnag issue but Bugsnags retention is quite limited and it a month or so the link won't be valid anymore.